### ISSUE TITLE
0x1a6 - "Not found pending summary, but this node has previously completed a summary" summary errors in telemetry

### ISSUE THREAD

**Issue by vladsud** [microsoft/FluidFramework#5599]
union Office_Fluid_FluidRuntime_*
| where Data_error contains "Not found pending summary, but this node has previously completed a summary"
| summarize count() by Data_runtimeVersion, Data_loaderVersion, Data_hostName, Data_docId

Gives me 249 errors

**Comment by vladsud** [comment]
This is now "**0x1a6**" error (so query above needs to be adjusted accordingly).
There are 9K hits (over 30 days) now in Kusto - it's substantial growth over last 2 months.
Can this be triaged soonish, please?

**Comment by vladsud** [comment]
Some observations looking at Data_containerId == "3a7bf686-dbe9-40d0-b79e-f655d8f7dd71"

1. **RefreshLatestSummaryGetSnapshot**_end fired, suggesting we saw an ack for snapshot. hasVersion = false, but there is no fluid:telemetry:OdspDriver:getVersions_end event in telemetry - code suggests it should be other way around.
2. fluid:telemetry:OdspDriver:**LastSummaryProposedHandleMismatch** fires by driver, suggesting driver & runtime disagree on what was the latest summary. I think it's a bug in driver, as we loaded from cache - TreesLatest_end fires after that message and driver is not aware that runtime knows about latest snapshot.
3. fluid:telemetry:Summarizer:Running:**GenerateSummary_end** fires
4. I do not see **SummaryOp** event, suggesting checkNotTimeout() check in summarizeCore() was triggered? We are only a second into session!
5. **MissingSummaryAckFoundByOps** event fires within 2 seconds of start of the session, while we are generating summary. Looks like some incoming op triggered it (though our own join session did not, maybe we got lucky with timing, it should be rare to have that kind of sequence of events).
6. We start new summary (instantly) and it also hits fluid:telemetry:OdspDriver:**LastSummaryProposedHandleMismatch** (I assume now due to # 4). This time hasVersion == true, but I see fluid:telemetry:OdspDriver:getVersions_end prior to it (part of that call) - similar to # 1, my reading of the code that reverse should happen.
7. This time we generate safe summayr, and get **0x1a6** right after fluid:telemetry:Summarizer:Running:**GenerateSummary_end** event. Exception is caught with no impact to flow.
8. There is fluid:telemetry:Summarizer:Running:SummaryOp following it. But session disconnects, so summarizer dies.
